mesonbuild/modules/python.py

```diff
@@ -477,7 +477,8 @@ def _get_win_pythonpath(self, name_or_path):
 
     @permittedKwargs(['required'])
     def find_installation(self, interpreter, state, args, kwargs):
-        disabled, required, feature = extract_required_kwarg(kwargs, state.subproject)
+        disabled, required, feature = extract_required_kwarg(kwargs, state.subproject,
+                feature_check=FeatureNew('Passing "feature" option to find_installation', '0.48.0'))
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->